servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Completable.java

```diff
@@ -1213,6 +1216,29 @@ public final Completable subscribeShareContext() {
         return new CompletableSubscribeShareContext(this);
     }
 
+    /**
+     * Creates a new {@link Completable} that ambiguates the result of this {@link Completable} with the passed
+     * {@code other} {@link Completable} such that whichever of them terminates first (successfully or with an error),
+     * the returned {@link Completable} will return that result.
+     * <p>
+     * From a sequential programming point of view this method is roughly equivalent to the following:
+     * <pre>{@code
+     *      for (Future<T> ft: futures) { // Provided Futures (analogous to the Completables here)
```

Comment:
@Scottmitch rite we generally do not do that in the documentation to avoid making the pseudo-code more complex to understand. 

@idelpivnitskiy 

That is correct, interestingly @colestanfield also told me the same offline. Although, I would like to avoid using specific `CompletableFuture` methods which is async to explain the imperative control-flow here. However, I can add an alias method here for `anyOf` which delegates to `amb`

Reply:
Generally, I don't like the `amb` name. I prefer more intuitive method names for discoverability, especially in an IDE where I'm looking for a method to do something specific to my problem. `Completable  anyOf(Iterable<Completable>)` to me also doesn't imply an order.

I'd suggest a better name would be `Completable firstOf(Iterable<Completable>)`, Completable firstSuccessOf(Iterable<Completable>), Completable firstFailedOf(Iterable<Completable>) or more generally `Completable findFirst(Iterable<Completable>, Predicate<Completable>)` . Implementing the last makes the others trivial helper operators.

Reply:
I agree the `amb` name isn't ideal. We try to balance "using names that are pre-established and recognizable in the reactive space" vs "using names that are more descriptive of the control flow". it can be difficult for folks to understand "which operator do I need" if all implementations have different names for the same operations. @NiteshKant - what is the precedent for this operator name vs choosing an alternative name?

Reply:
Thanks all for the input here. I already added `anyOf` as an alias as there isn’t a clear preference for a name across communities here (as is the case with most operators). Other `anyOf` variants are useful but not required for my related work ATM and we can add them incrementally later. So I will defer them for now.